app/scripts/modules/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@spinnaker/core",
-  "version": "0.0.148",
+  "version": "0.0.149",
```

Comment:
best part of pr

Reply:
I went back and forth on it before settling on this